client/meteor-analytics.js

```diff
@@ -150,3 +150,10 @@ Meteor.startup(function () {
     Tracker.autorun(trackLogins);
   }
 });
+
+
+analytics = this.analytics;
+try {
```

Comment:
Well it would make sense if it doesn't do anything. But I was more just
asking out of interest.
On 22 May 2016 11:44, "Mickael Lecoq" <notifications@github.com> wrote:

> In client/meteor-analytics.js
> <https://github.com/okgrow/analytics/pull/107#discussion_r64146373>:
>
> > @@ -150,3 +150,10 @@ Meteor.startup(function () {
> >      Tracker.autorun(trackLogins);
> >    }
> >  });
> > +
> > +
> > +analytics = this.analytics;
> > +try {
>
> @elie222 <https://github.com/elie222> I can remove it if you prefer ;)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/okgrow/analytics/pull/107/files/7ced6f3af869c675b0a1351d093bf2c978a17a53#r64146373>
>